app/assets/src/components/views/samples/CollectionModal.jsx

```diff
@@ -12,6 +12,10 @@ import Modal from "~ui/containers/Modal";
 import Notification from "~ui/notifications/Notification";
 import cs from "./collection_modal.scss";
 
+/**
+ * NOTE: "Collections" were an unrealized generalization of the background concept.
+ * For the time being, a collection is equivalent to a background.
+ */
 class CollectionModal extends React.Component {
```

Comment:
I think we should change the name of the React component while we're at it (and all the other references to a "collection modal"), to minimize confusion.

Reply:
Sorry for the late reply. I'm not sure of the status of the "collections" project or else I would be happy to rename as well. This PR is intended to be a minimal change of wording to avoid confusing users about the nonexistent collections feature. @happyimadesignr is there any chance collections will come back?